fleet_notifications/script_args/args.py

```
@@ -0,0 +1,129 @@
+
+def _new_arg_parser(script_description: str) -> argparse.ArgumentParser:
+    return argparse.ArgumentParser(description=script_description)
+
+
+def load_config_file(path: str) -> dict[str, Any]:
+    try:
+        with open(path) as config_file:
+            config = json.load(config_file)
+            return config
+    except FileNotFoundError:
+        raise ConfigFileNotFound(f"Could not load config file from path '{path}'.")
```

Ruff (0.8.0)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)